_posts/2022-09-06-conscryptmode.md

```diff
@@ -0,0 +1,19 @@
+Google Summer of Code (GSoC) is a program where external participants can contribute to an open source project over a few months. We learn new computer science concepts, how to work on an open source repository, and create real contributions to Google code!
+
+My project was `Switching Robolectric from BouncyCastle to Conscrypt as the default security provider`. Robolectric was using [BouncyCastle](https://www.bouncycastle.org/) as the Java Cryptography Extension (JCE) security provider. But after the introduction of Android P, Android switched to using Google Conscrypt as the security provider. 
+To be more consistent with Android, Robolectric needed to be updated to use Conscrypt as the default security provider.
+ 
+I was already well familiar with the technologies used in the project such as Java, Kotlin, Android Studio, Git and GitHub. I was interested in Android Developement and I was looking for an android project that uses Java as the primary language. I found Robolectric as a perfect project to contribute according to my skills and I was ready to learn about testing concepts since Robolectric is a popular unit testing framework for Android. 
```

Comment:
nit: `I found Robolectric to be a perfect project to contribute to based on my skills, and I was ready to learn...`